src/EditorFeatures/TestUtilities/QuickInfo/AbstractSemanticQuickInfoSourceTests.cs

```diff
@@ -160,132 +160,95 @@ protected string Lines(params string[] lines)
             return null;
         }
 
-        private static void AssertTextAndClassifications(string expectedText, Tuple<string, string>[] expectedClassifications, IDeferredQuickInfoContent actualContent)
+        internal Action<QuickInfoItem> SymbolGlyph(Glyph expectedGlyph)
         {
-            var actualClassifications = ((ClassifiableDeferredContent)actualContent).ClassifiableContent;
-
-            ClassificationTestHelper.VerifyTextAndClassifications(expectedText, expectedClassifications, actualClassifications);
+            return qi =>
+            {
+                Assert.True(qi.Tags.GetGlyphs().Contains(expectedGlyph));
```

Comment:
Consider Assert.Contains to give a better message if it fails.

Reply:
fixed